### DISCUSSION TITLE
how to scale learning rate?

### DISCUSSION THREAD

**Discussion by fcakyon** [open-mmlab/mmyolo#295] [Q&A] [answered]
The original YOLOv5 repo calculates the learning rate and schedule automatically based on the given epoch number, batch size, and GPU count.

In MMYOLO, default lr is 0.01, but I assume it is calculated for 8 batch size x 8 GPU. 

How should I scale it in my fine-tuning task when using N GPUs with M batch size per GPU?

**Comment by fcakyon** [comment]
Do you have any suggestions? @hhaAndroid

**Comment by RangeKing** [comment, marked as answer]
Thank you for your attention. Related PR https://github.com/open-mmlab/mmyolo/pull/486.
